tools/tck-api/src/test/scala/org/opencypher/tools/tck/TckTest.scala

```diff
@@ -52,7 +52,7 @@ class TckTest extends AnyFunSpec with Assertions {
     }
   }
 
-  private object FakeGraph extends Graph with ProcedureSupport {
+  private object FakeGraph extends Graph with ProcedureSupport with CsvFileCreationSupport {
```

Comment:
I would test the whole roundtrip (multiple comments):

```suggestion
  private object FakeGraph extends Graph with ProcedureSupport with CsvFileCreationSupport {
    var cvsData = CypherValueRecords.empty
```